src/components/expanded-state/custom-gas/FeesPanelTabs.tsx

```diff
@@ -42,7 +42,7 @@ const TabPill = ({
       onPress={handleOnPress}
       scaleTo={0.8}
       testID={testID}
-      paddingHorizontal="5px"
+      paddingHorizontal="5px (Deprecated)"
```

Comment:
Same as above, this won't be a diff once https://github.com/rainbow-me/rainbow/pull/4157 is merged.